tv_apps/smart-home/js/edit.js

```diff
@@ -352,7 +352,11 @@
     },
 
     deleteCard: function(scrollable, nodeElem) {
-      document.activeElement.blur();
+      // we have to move focus to another element to prevent user press enter
+      // to trigger unexpected behavior. We cannot use blur() to do it because
+      // it makes smart-home without focus. In this case, other apps can
+      // request focus from smart-home which makes focus lost, bug 1161940.
+      document.getElementById('main-section').focus();
```

Comment:
Do we need it here?
`this.modalDialog` is an internal module without calling `contextmenu` now, so I guess adding it to `onVisibilityChange` is just enough. How do you think?